src/conn/conn_open.c

```diff
@@ -139,6 +145,9 @@ __wti_connection_close(WT_CONNECTION_IMPL *conn)
     /* Disconnect from shared cache - must be before cache destroy. */
     WT_TRET(__wti_conn_cache_pool_destroy(session));
 
+    /* Destroy Eviction. */
+    WT_TRET(__wti_evict_destroy(session));
+
     /* Discard the cache. */
     WT_TRET(__wti_cache_destroy(session));
```

Comment:
No action needed: We've changed code order here - we used to perform the `/* The cache should be empty at this point.  Complain if not. */` checks first, but now they take place after we destroy the locks and shutdown the internal session in `__wti_evict_destroy`.

The change looks safe. I'm just flagging it as a function change to the code.